### PR TITLE
fix(restInterface.js): Hide metrics link unless users are ADMIN and METR...

### PR DESCRIPTION
**Pull request by SOAP1914**
...ICS

issue-490

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
     "grunt-contrib-less": "~0.11.0",
     "grunt-contrib-uglify": "^0.2.7",
     "grunt-contrib-watch": "^0.4.4",
-    "grunt-contrib-yuidoc": "^0.5.2",
+    "grunt-contrib-yuidoc": "0.5.2",
```
Comment (arw180) [inline package.json]: what was the reasoning for this?

Comment (SOAP1914) [inline package.json]: it was killing file on the build in newer versions

Comment (arw180) [inline package.json]: I don't know what you mean by that. The `^` will automatically upgrade the package - `grunt-contrib-yuidoc`, in this case - to the latest in the `0.x.x` range. Was it upgrading to a specific version of the yuidoc package that was breaking something? Either way, this change locks the version of the yuidoc package at exactly `0.5.2`, which is almost certainly not what we want. Did you try `~0.5.2`, which will take the latest 0.5.x release, but skip 0.6.x (and beyond)?

See http://stackoverflow.com/questions/22343224/difference-between-tilde-and-caret-in-package-json for details

**Comment by arw180** [comment]
This looks good now, I'll go ahead and merge it in

**Pull request merged**